README.md

```diff
@@ -54,12 +54,20 @@ icrc1_metadata : () -> (vec { record { text; Value } }) query;
 
 ### icrc1_total_supply
 
-Returns the total token supply.
+Returns the total number of tokens on all accounts except for the minting account.
 
 ```
 icrc1_total_supply : () -> (nat) query;
 ```
 
+### icrc1_minting_account
+
+Returns the [minting account](#minting_account) for this ledger.
+
+```
+icrc1_minting_account : () -> (Account) query;
```

Comment:
How is the minting account set? Should the service have a parameter for the minting account?

Reply:
We discussed this during the WG call today, I understand why it's left up to the implementation how the minting account is set.